lib/jekyll/renderer.rb

```diff
@@ -185,10 +219,21 @@ def converter_output_ext
       end
     end
 
+    private
     def output_exts
       @output_exts ||= converters.map do |c|
         c.output_ext(document.extname)
       end.compact
     end
+
+    private
+    def path
+      document.path
+    end
+
+    private
+    def relative_path
+      document.relative_path
+    end
```

Comment:
These two methods are superfluous – there is no reason to make a method here, and `Renderer#relative_path` doesn't make sense because the Renderer doesn't have any concept of path.